src/app/consensus_test/consensus_test.ml

```diff
@@ -0,0 +1,229 @@
+open Core_kernel
+open Async
+
+module BlockFileOutput = struct
```

Comment:
Style NIT: Use `Snake_case` for module names to follow the jane street style guide (which we follow with some adaptations). https://opensource.janestreet.com/standards/